demo/demo.md

```diff
@@ -98,11 +98,48 @@ This example illustrates a mainline nonstop with a next day arrival or departure
   arrivalTime="2022-06-01T06:45:00-07:00"
   arrivalStation="SEA">
   <auro-flightline></auro-flightline>
-</auro-flight>
-```
-<!-- AURO-GENERATED-CONTENT:END -->
-</auro-accordion>
-
+</auro-flight>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+## Mainline -1 day arrival/departure
+
+This example illustrates a mainline nonstop with a -1 day arrival or departure from KOA to SEA
+
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/nextDay.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/nextDay.html -->
+  <auro-flight
+    flights='["AS 880"]'
+    duration="350"
+    departureTime="2022-05-31T22:55:00-10:00"
+    departureStation="KOA"
+    arrivalTime="2022-05-30T07:45:00-07:00"
+    arrivalStation="SEA">
+    <auro-flightline></auro-flightline>
+  </auro-flight>
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion alignRight>
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/nextDay.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/nextDay.html -->
+
+```html
+<auro-flight
+  flights='["AS 880"]'
+  duration="350"
+  departureTime="2022-05-31T22:55:00-10:00"
```

Comment:
There is no example file to support this inclusion. When a new build is ran, all of this new content is deleted. 

Please add all content to a file in `./apiExamples` and map reference in `./docs/partials/demo.md` file to build the `./demo/demo.md` file you edited.

Reply:
oh, sorry I will fix ASAP, thank you @blackfalcon.